mobile/src/main/java/org/openhab/habdroid/ui/activity/PageConnectionHolderFragment.kt

```diff
@@ -423,12 +423,14 @@ class PageConnectionHolderFragment : Fragment(), CoroutineScope {
                         callback.onWidgetUpdated(url, updatedWidget)
                         return
                     }
+                } else if(lastWidgetList != null) {
```

Comment:
```suggestion
                } else if (lastWidgetList != null) {
```

(code linter will yell about this)

Reply:
@mueller-ma We don't run the linter as part of PR actions?

Reply:
No, we used https://github.com/apps/stickler-ci, but it seems that they shut down. I think we lost it when migrating to Kotlin.

I opened https://github.com/openhab/openhab-android/issues/3640